modules/dmrpp_module/SuperChunk.cc

```diff
@@ -74,6 +76,8 @@ void process_one_chunk(shared_ptr<Chunk> chunk, DmrppArray *array, const vector<
 {
     BESDEBUG(SUPER_CHUNK_MODULE, prolog << "BEGIN" << endl );
 
+    // TODO If this is a SuperChunk, hasn't the data been read by SuperChunk::Retrieve_data()?
```

Comment:
I'm don't know what the answer is but it seems like we should resolve this question now or write a ticket to do so.

Reply:
[HYRAX-713](https://bugs.earthdata.nasa.gov/browse/HYRAX-713)